ingestion/functions/parsing/cuba/cuba.py

```diff
@@ -161,16 +161,23 @@ def parse_cases(raw_data_file, source_id, source_url):
         # First make dict mapping code names of diagnostic and treament centers
         # to actual locations
         hospital_map = {}
-        for centre_type in ['centros_aislamiento', 'centros_diagnostico']:
-            for centre in json_data[centre_type]:
-                hospital_map[centre] = json_data[centre_type][centre]['nombre'] + \
-                    ", " + json_data[centre_type][centre]['provincia']
+        try:
+            for centre_type in ['centros_aislamiento', 'centros_diagnostico']:
+                for centre in json_data[centre_type]:
+                    hospital_map[centre] = json_data[centre_type][centre]['nombre'] + \
+                        ", " + json_data[centre_type][centre]['provincia']
+        except:
```

Comment:
use except KeyError here - in case some other exception is raised. Also good to use logging.info to print a message saying that key was not found

Reply:
New commit [6c96b56] deals with this.